FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/DataConverter/DefaultSchemaConverter.cs

```diff
@@ -125,9 +126,9 @@ private JArray ProcessArrayObject(JToken arrayItem, FhirParquetSchemaNode schema
         {
             if (arrayItem is not JArray fhirArrayObject)
             {
-                _diagnosticLogger.LogError($"Current FHIR object is not a valid JArray: {schemaNode.GetNodePath()}.");
-                _logger.LogInformation($"Current FHIR object is not a valid JArray: {schemaNode.GetNodePath()}.");
-                throw new ParquetDataProcessorException($"Current FHIR object is not a valid JArray: {schemaNode.GetNodePath()}.");
+                _diagnosticLogger.LogError($"Current FHIR object is not a valid JArray: {arrayItem.Path}.");
+                _logger.LogInformation($"Current FHIR object is not a valid JArray: {arrayItem.Path}.");
```

Comment:
When we use `item.Path` and when we use `item` directly in logging message?

Reply:
That actually is a bug, fixed, thanks.